config/config.exs

```diff
@@ -2,4 +2,5 @@ use Mix.Config
 
 if Mix.env == :test do
   config :ex_unit, :assert_receive_timeout, 1000
+  config :postgrex, :pgpass, "test/support/pgpass"
```

Comment:
Application config does not work well when there are multiple postgresql backends. I think we should use System.get_env `PGPASSFILE`. This can be an option `passfile` to `start_link`.